common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signin/SignInInitiateApiResponse.kt

```diff
@@ -39,14 +39,13 @@ import java.net.HttpURLConnection
  */
 data class SignInInitiateApiResponse(
     @Expose override var statusCode: Int,
-    @SerializedName("credential_token") val credentialToken: String?,
+    @SerializedName("continuation_token") val continuationToken: String?,
     @Expose @SerializedName("challenge_type") val challengeType: String?,
```

Comment:
actually... is this field correct? Does /initiate return this field?

Reply:
Yes if the challenge types sent in the request are not supported by the server /initiate returns redirect with this field

Reply:
Ofcourse :)